confluence/client.py

```diff
@@ -84,6 +84,12 @@ def _get_paged_results(self, item_type, path, params, expand):
             for result in search_results['results']:
                 yield item_type(result)
 
+    def put_content(self, content, content_type = 'page'):
+        # type: (Page, str) -> None
+        url = f'{self._api_base}/content/{content.id}'
```

Comment:
I decided since this is on pypi I'd better support more than just 3.6. Can you switch this to using .format instead.